data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+ACME_BASE=/var/lib/acme
+mkdir -p ${ACME_BASE}/acme/private
+
+restart_containers(){
+	for container in $*; do
+		curl -X POST \
+			--unix-socket /var/run/docker.sock \
+			"http:/containers/${container}/restart"
```

Comment:
Well, this was my first thought too. But it's consistently written in the [docker documentation](https://docs.docker.com/engine/api/get-started).